pyproject.toml

```diff
@@ -1,4 +1,170 @@
 [tool.black]
 line-length = 100
-target-version = ['py38']
+target-version = ["py38"]
 skip-string-normalization = true
+
+[tool.isort]
+include_trailing_comma = true
+line_length = 100
+multi_line_output = 3
+
+[tool.poetry]
+name = "toucan-connectors"
+version = "3.8.0"
+description = "Toucan Toco Connectors"
+authors = ["Toucan Toco <dev@toucantoco.com>"]
+license = "BSD"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+# <3.11 required by awswrangler
+python = ">=3.8,<3.11"
+Authlib = "^1.0.1"
```

Comment:
IMO, we should have for all our dependencies  `>=` instead of `^`, related discussion on laputa with @fspot : https://github.com/ToucanToco/laputa/pull/5135#discussion_r879380638

Reply:
I disagree, especially for stable libs such as `Authlib` or `psycopg2`, were suddenly having a new major version in production images is definitely not something we want. But you're right for 0ver, I'll update everything in 0ver to be `>=0.x,<1.0`